sound/soc/sdca/sdca_functions.c

```diff
@@ -1220,6 +1221,98 @@ static int find_sdca_entity_ge(struct device *dev,
 	return -EINVAL;
 }
 
+static int
+find_sdca_entity_hide(struct device *dev, struct fwnode_handle *function_node,
+					struct fwnode_handle *entity_node,
+					struct sdca_entity *entity)
+{
+	struct sdca_entity_hide *hide = &entity->hide;
+	struct hid_descriptor hid_desc;
+	unsigned int delay, af_list[SDCA_MAX_FUNCTION_COUNT];
+	int nval, ret;
+	unsigned char *report_desc = NULL;
+
+	ret = fwnode_property_read_u32(entity_node,
+			"mipi-sdca-RxUMP-ownership-transition-maxdelay", &delay);
+	if (!ret)
+		hide->max_delay = delay;
+
+	nval = fwnode_property_count_u32(entity_node, "mipi-sdca-HIDTx-supported-report-ids");
+	if (nval > 0) {
+		hide->num_hidtx_ids = nval;
+		hide->hidtx_ids = devm_kcalloc(dev, hide->num_hidtx_ids,
+							sizeof(*hide->hidtx_ids), GFP_KERNEL);
+		if (!hide->hidtx_ids)
+			return -ENOMEM;
+
+		ret = fwnode_property_read_u32_array(entity_node,
+						"mipi-sdca-HIDTx-supported-report-ids",
+						hide->hidtx_ids,
+						hide->num_hidtx_ids);
+		if (ret < 0)
+			return ret;
+	} else {
+		hide->hidtx_ids = NULL;
+	}
```

Comment:
Don't need to clear this here, it will already be zeroed.

Reply:
will fix